frontend/components/common/Select.vue

```diff
@@ -108,7 +112,11 @@ export default Vue.extend({
     watch: {
         localVisible: {
             handler(newVisible) {
-                this.$emit('update:visible', newVisible);
+                if (this.disabled) {
+                    this.localVisible = false;
+                } else {
```

Comment:
Can you disable the actual selects / modals instead of silently skipping emit here? I would even say that we should add isLoading animation to the network label in the header (so probably you should pass it as `isLoading` to the `Select`)

Reply:
The popover / tooltip doesn't seem to have a disabled property, so this was my attempt to do that

For the animation, I already added a loader when `isChangingNetwork` is true which replaces the network icon for that time.

Reply:
> The popover / tooltip doesn't seem to have a disabled property

- If no content is provided they are not shown
- You can disable all buttons within the popover
- You can replace the network label with `Loading...` which doesn't have popover over it at all

I would go with the last option, but silently skipping user action is an anti-pattern

Reply:
Yep, the last one sounds the best to me too. Will add that!